docs/release-notes/mongoid-7.4.txt

```diff
@@ -390,12 +341,64 @@ all scopes are cleared:
   by default in Mongoid 8.0.
 
 
-Demongoize Values Returned from Pluck and Distinct
---------------------------------------------------
+Changes to ``distinct`` and ``pluck``
+-------------------------------------
+
+Respect Field Aliases In Embedded Documents When Using ``distinct`` and ``pluck``
+`````````````````````````````````````````````````````````````````````````````````
+
+When ``distinct`` and ``pluck`` are used with aliased fields in embedded
+documents, the aliases can be expanded if the ``Mongoid.broken_alias_handling``
+option is set to ``false``. By default, for backwards compatibility, in
+Mongoid 7.4 this option is set to true, yielding Mongoid 7.3 and earlier
+behavior. Given the following definitions:
+
+.. code-block:: ruby
+
+  class Band
+    include Mongoid::Document
+    embeds_many :managers
+  end
+
+  class Manager
+    include Mongoid::Document
+    embedded_in :band
+
+    field :name, as: :n
+  end
+
+Mongoid 7.4 behavior with ``Mongoid.broken_alias_handling`` set to ``false``:
+
+.. code-block:: ruby
+
+  # Expands out to "managers.name" in the query:
+  Band.distinct('managers.n')
+  Band.pluck('managers.n')
+
+Mongoid 7.3 and 7.4 with ``Mongoid.broken_alias_handling`` set to ``true`` behavior:
+
+.. code-block:: ruby
+
+  # Sends "managers.n" without expanding the alias:
+  Band.distinct('managers.n')
+  Band.pluck('managers.n')
```

Comment:
And here I think the actual usage would be "managers.name" as well (but like you said without alias expansion).